packages/js-moi-providers/__tests__/jsonrpc-provider.test.ts

```diff
@@ -241,4 +243,89 @@ describe("Test JsonRpcProvider Query Calls", () => {
         expect(accounts.length).toBeGreaterThanOrEqual(2)
       });
     });
+
+    describe("getNewTesseractFilter", () => {
+      it("should return the filter object containing the filter id", async () => {
+        const filter = await provider.getNewTesseractFilter();
+
+        expect(filter).toBeDefined();
+        expect(filter).toHaveProperty("id");
+        expect(typeof filter.id).toBe("string");
+      });
+    });
+
+    describe("getNewTesseractsByAccountFilter", () => {
+      it("should return a filter object containing filter id", async () => {
+        const filter = await provider.getNewTesseractsByAccountFilter(address);
+
+        expect(filter).toBeDefined();
+        expect(filter).toHaveProperty("id");
+        expect(typeof filter.id).toBe("string");
+      })
+    });
+
+    describe("getPendingInteractionFilter", () => {
+      it("should return a filter object containing filter id", async () => {
+        const filter = await provider.getPendingInteractionFilter();
+
+        expect(filter).toBeDefined();
+        expect(filter).toHaveProperty("id");
+        expect(typeof filter.id).toBe("string");
+      })
+    });
+
+    describe("removeFilter", () => {
+      it("should a return a object containing status of removal", async () => {
+        const filter = await provider.getNewTesseractsByAccountFilter(address);
+
+        const result = await provider.removeFilter(filter);
+
+        expect(result).toBeDefined();
+        expect(result).toHaveProperty("status");
+        expect(result.status).toBe(true);
+      })
+
+      it("should return a object with status false when filter is deleted or not exist", async () => {
```

Comment:
test case name can be improved.

Reply:
6144a3467ccfbaf92c4f59eeb12fffd747006319